doc/guides/misc/managing-a-release.md

```diff
@@ -194,9 +194,9 @@ A point release for a critical issue is best limited to a small number of change
 | Release manager tool | https://github.com/RIOT-OS/RIOT-release-manager |
 | Test automation helper scripts | https://github.com/RIOT-OS/Release-Specs?tab=readme-ov-file#pytest-runner |
 | Pending changes to test automation helper scripts | https://github.com/jia200x/Release-Specs/pulls |
-| Backport automation script | {RIOTBASE}/dist/tools/backport_pr |
+| Backport automation script | https://github.com/RIOT-OS/RIOT/blob/master/dist/tools/backport_pr |
 | Release statistics wiki | https://github.com/RIOT-OS/RIOT/wiki/release-statistics |
-| Release stats script | {RIOTBASE}/dist/tools/release-stats |
+| Release stats script | https://github.com/RIOT-OS/RIOT/blob/master/dist/tools/release-stats |
```

Comment:
Wait what, did we do Macro Magic there before?

Reply:
That link was not working and this is the only place, so I removed it.

Reply:
ahhh